packages/@unimodules/react-native-adapter/package.json

```diff
@@ -34,7 +34,7 @@
   },
   "peerDependencies": {
     "react-native": "*",
-    "react-native-web": "~0.11"
+    "react-native-web": "~0.12"
```

Comment:
`0.12` is not a valid semver, I'd use `~0.12.0` instead.

Reply:
Agreed. Specify the patch to specify the minimum version.